docs/ru/assemble_4.md

```diff
@@ -0,0 +1,336 @@
+# Сборка Клевера 4
+
+<img src="../assets/clever-4-blow.png" class="zoom center" width=400>
+
+## Сборка основы для рамы
+
+1. Совместите 2 карбоновые пластины жесткости, используя центральные пазы.
+2. Используя пазы, установите сверху карбоновую центральную деку.
+
+    <center>
+        <img src="../assets/4/01.png" width=300 class="zoom border">
+        <img src="../assets/4/02.png" width=300 class="zoom border">
+    </center>
+
+3. Стяните конструкцию с помощью винтов М3х8 и стальных гаек, установленных в пазах пластин.
+
+    <center>
+        <img src="../assets/4/03.png" width=300 class="zoom border">
+        <img src="../assets/4/03_1.png" width=300 class="zoom border">
+    </center>
+
+## Установка моторов
+
+1. Распакуйте моторы.
+2. Используя ножницы, укоротите провода на моторах:
```

Comment:
Do we really wish our users to use scissors? I'm putting something more appropriate (wire strippers) in the English instruction.